product_bom_revision/views/mrp_view.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
```

Comment:
The xml version is no longer needed.

Reply:
The encoding or the version because all the modules in v13 has the version in the xml files

Reply:
That whole line is unnecessary since version 11 I believe but it doesn't hurt to have it in there. It doesn't affect anything whether it's there or not so that's why I approved this anyway.